ui/src/ducks/inventory/index.js

```diff
@@ -1,17 +1,91 @@
+import axios from 'axios'
 import { createAction, handleActions } from 'redux-actions'
+import { openSuccess } from '../alerts'
 
 const actions = {
   INVENTORY_GET_ALL: 'inventory/get_all',
+  INVENTORY_PENDING: 'inventory/pending',
+  INVENTORY_CREATE: 'inventory/create',
+  INVENTORY_UPDATE: 'inventory/update',
+  INVENTORY_DELETE: 'inventory/delete',
+  INVENTORY_REFRESH: 'inventory/refresh'
 }
 
 export let defaultState = {
-  all: []
+  all: [],
+  fetched: false,
 }
 
 export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => {
-  //TODO
+  (dispatch, getState, config) => axios
+    .get('${config.restAPIURL}/inventory')
+    .then((suc) => {
+      openSuccess()
+      dispatch(refreshInventory(suc.data))
+    })
 })
+export const createInventory = createAction(actions.INVENTORY_CREATE, (inventory) => {
+  (dispatch, getState, config) => axios
+    .post('${config.restAPIURL}/inventory', inventory)
+    .then((suc) => {
+      openSuccess()
+      const invs = []
+      //adds all the inventory not matching the created one to the array
+      getState().inventories.all.forEach(inv => {
+        if (inv.id !== suc.data.id) {
+          invs.push(inv)
+        }
+      })
+      //adds created inventory to the array
+      invs.push(suc.data)
+      //refresh with contents of the array
+      dispatch(refreshInventory(invs))
+    })
+})
```

Comment:
When using an arrow function ( => ), if you have curly braces you also need to use the `return` keyword. If you omit the curly braces, you can omit the return keyword.